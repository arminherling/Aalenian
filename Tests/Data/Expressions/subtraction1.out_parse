AssignmentStatement: {
    Left: {
        DiscardLiteral: _
    }
    Right: {
        BinaryExpression: {
            Left: {
                NumberLiteral: 1
            }
            Operation: Subtraction
            Right: {
                NumberLiteral: 2
            }
        }
    }
}
