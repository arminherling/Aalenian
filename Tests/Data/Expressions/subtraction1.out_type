TypedAssignmentStatement: {
    Type: i32
    Left: {
        Discard: {
            Type: _
        }
    }
    Right: {
        TypedSubtractionExpression: {
            Type: i32
            Left: {
                I32Value: {
                    Type: i32
                    Value: 1
                }
            }
            Right: {
                I32Value: {
                    Type: i32
                    Value: 2
                }
            }
        }
    }
}
