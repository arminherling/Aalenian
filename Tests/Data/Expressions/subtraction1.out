AssignmentStatement: {
    Left: {
        Discard: _
    }
    Right: {
        BinaryExpression: {
            Left: {
                Number: 1
            }
            Operation: Subtraction
            Right: {
                Number: 2
            }
        }
    }
}
